ucsb-cs156-f23/proj-happycows-f23-5pm-3
# Kelly - set default values for create commons page

## Overview
In this PR, changes to the frontend have been made such that the Create Commons form autopopulates with default values specified in the applications.properties file. The form is still editable and successfully creates a Commons upon submission. The toast message has been updated to encompass the user input values for all fields as a sanity check. The starting date defaults to today if not specified in the initialization values.

Dokku: https://happycows-madisonlong1.dokku-03.cs.ucsb.edu/

## Screenshots
<img width="1430" alt="HappyCows Create Commons" src="https://github.com/ucsb-cs156-f23/proj-happycows-f23-5pm-3/assets/26288908/b92da2b3-3423-47ac-b7ee-996d4be263d1">

Above we see the form populated with default values.

## Feedback Request
- Open to any general feedback/comments

## Validation
1. Download this branch.
2. Run the project.
3. Log in as an admin user.
4. On the top navbar go to Admin > Create Commons.
5. See the default input values populated in the form.
6. Modify the default values using the .env file. Save and see the changes.

## Tests

## Linked Issues

Closes #10, #18
